src/main/java/seedu/address/logic/parser/FindPropertyCommandParser.java

```diff
@@ -28,6 +41,7 @@
      * @throws ParseException if the user input does not conform the expected format
      */
     public FindPropertyCommand parse(String args) throws ParseException {
```

Comment:
I agree with david that you should probably try to use `ArgumentTokenizer` here. This function has to be modularized one way or another because it is way too long and not reader-friendly now